src/cmd/vm.rs

```diff
@@ -54,8 +54,8 @@ pub fn run(args: &Args) -> Result<()> {
 /// run first time setup, installs necessary dependencies
 #[argh(subcommand, name = "setup")]
 pub struct ArgsSetup {
-    /// path to dir where betty.sh exists. If omitted, current directory will be
-    /// used.
+    /// path to android repo dir where betty.sh exists. If omitted, current
```

Comment:
maybe removing `where betty.sh exists` is better.
also, `android repo dir` can be replaced with `the android source checkout` or something, since `repo` has multiple meaning in our context... (repo as a command / repo which is a short form of repository / ...)

Reply:
maybe the term `ARC` stands for `Android Repository Checkout` , not only the `Android Runtime on ChromeOS` 😉